frontend/public/actions/k8s.ts

```diff
@@ -1,71 +1,43 @@
 import * as _ from 'lodash-es';
 import { Dispatch } from 'react-redux';
 import { ActionType as Action, action } from 'typesafe-actions';
-import { FilterValue, getImpersonate } from '@console/dynamic-plugin-sdk';
 import { checkAccess } from '@console/internal/components/utils/rbac';
 
 import {
   cacheResources,
   getResources as getResources_,
   DiscoveryResources,
 } from '../module/k8s/get-resources';
-import {
-  k8sList,
-  k8sWatch,
-  k8sGet,
-  referenceForModel,
-  K8sResourceKind,
-  K8sKind,
-  fetchSwagger,
-} from '../module/k8s';
+import { K8sResourceKind, fetchSwagger } from '../module/k8s';
 import { makeReduxID } from '../components/utils/k8s-watcher';
 import { CustomResourceDefinitionModel } from '../models';
+import { watchK8sList } from '@console/dynamic-plugin-sdk/src/app/k8s/actions/k8s';
+
+export {
+  watchK8sObject,
+  watchK8sList,
+  stopK8sWatch,
+} from '@console/dynamic-plugin-sdk/src/app/k8s/actions/k8s';
 
 export enum ActionType {
   ReceivedResources = 'resources',
   GetResourcesInFlight = 'getResourcesInFlight',
-
-  StartWatchK8sObject = 'startWatchK8sObject',
-  StartWatchK8sList = 'startWatchK8sList',
-  StopWatchK8s = 'stopWatchK8s',
-  ModifyObject = 'modifyObject',
-
-  Loaded = 'loaded',
-  Errored = 'errored',
-
-  BulkAddToList = 'bulkAddToList',
-  FilterList = 'filterList',
-  UpdateListFromWS = 'updateListFromWS',
 }
 
 export const API_DISCOVERY_POLL_INTERVAL = 60000;
-const WS = {} as { [id: string]: WebSocket & any };
-const POLLs = {};
-const REF_COUNTS = {};
 
-const nop = () => {};
-const paginationLimit = 250;
+const POLLs = {};
```

Comment:
Isn't `POLLs` an internal module state that was moved over, along with associated logic, to `packages/console-dynamic-plugin-sdk/src/app/k8s/actions/k8s.ts`? (Do we still need it here?)

Reply:
`startAPIDiscovery` (where this is used) is part of the bootstrap of the app... I think this has value to be moved into the SDK too... maybe this is another PR? Not sure it matters if it's part of this one or not, but it should be moved too.

Reply:
Agreed on moving `startAPIDiscovery` as a follow-up change, thanks for clarification.

Reply:
For clarity - yes; we were planning on trying to move startAPIDiscovery in another PR - it utilizes `receivedResources` which has some redux-antipatterns we will need to refactor.